nixos/modules/services/web-apps/weblate.nix

```
@@ -166,6 +168,15 @@ in
         type = lib.types.path;
       };
 
+      configurePostgresql = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        description = ''
+          Whether to enable and configure a local PostgreSQL server by creating a user and database for weblate.
+          The default `settings` reference this database, if you disable this option you must provide a database URL.
```

```suggestion
          The default `settings` reference this database, if you disable this option you must provide a database URL in `extraConfig`.
```

This should hint the user on where to put the database URL.